### DISCUSSION TITLE
Quasar input field missed some characters

### DISCUSSION THREAD

**Discussion by harshalizode** [elimintz/justpy#694] [Q&A] [answered]
Hello Everyone!,

I am using the QInput in my JustPy application. When I type fast on any input field and submit it then, it missed some characters and return the incorrect string in backend.

When I copy paste the values and submit it then, it will return the empty string in backend.

How should we fix it?
Thanks.

**Comment by WolfgangFahl** [comment]
This is a known bug. You might want to try nicegui which doesn't have the problem.

**Reply by harshalizode** [reply]
Thank you for your response,  but we can't change the framework as we built our application using justpy.

**Comment by WolfgangFahl** [comment]
You can still ask in the nicegui discussion how they fixed the problem and suggest a patch for justpy.

**Reply by falkoschindler** [reply]
@WolfgangFahl Thanks for mentioning [NiceGUI](https://nicegui.io/)!

While JustPy was indeed an inspiration for NiceGUI 0.9, it's important to clarify that NiceGUI is not a direct fork. JustPy has been [removed and replaced with a completely new implementation](https://github.com/zauberzeug/nicegui/commit/f6194c03ec0917b9932dc661145f68b4d49e1f32) in 1.0. This includes a new update and event system, which has been continuously improved in subsequent updates, especially for rapid user input (https://github.com/zauberzeug/nicegui/issues/287, https://github.com/zauberzeug/nicegui/pull/306). As such, we didn't actually "fix" JustPy and, therefore, are unable to offer support or solutions specific to JustPy.

**Reply by harshalizode** [reply]
Thank you for your response,  I will check it 🙂.

**Comment by elimintz** [comment]
Hi,

Please take a look at https://justpy.io/quasar_tutorial/quasar_components/#qinputchange-and-qinputblur-disabling-the-input-event
This may be a solution to your problem.

**Reply by harshalizode** [reply]
Thank you for your response

**Reply by harshalizode** [reply]
Hi,  I go through that link, but the QInputBlur component can't satisfy our goal,  can we able to do the delay of 250 to 300ms on each input event in QInput component,  can we able to override it? 
Thanks.

**Comment by harshalizode** [comment, marked as answer]
Hi everyone to solve this issue we need to set argument debounce with the amount in milliseconds. 
This argument helps me to solve this problem. 
Thanks, everyone for helping to solve this problem.

**Comment by harshalizode** [comment]
Hello guys,  This issue solved after the R & D
